protocols/protocol/cloudstate/crdt.proto

```diff
@@ -92,8 +118,20 @@ message CrdtStreamIn {
     }
```

Comment:
"Delete the entity. May be sent at any time. The user function should **clear its state** when it receives this."
What does "clear its state" mean? deleted? Is the proxy allowed to send ever again a message for this entity?

Reply:
Regarding a deleted CRDT entity and the spec saying:
> Delete the entity. May be sent at any time. The user function should clear its state when it receives this.
> A proxy may decide to terminate the stream after sending this.

the Go Support clears any state for an entity and in theory, would allow a new entity to be initialised on that cleared stream. The stream, therefore, will not be terminated for a deleted entity.